storage/index.go

```diff
@@ -166,6 +166,11 @@ func newNamespaceIndexWithOptions(
 		return nil, err
 	}
 
+	scope := opts.InstrumentOptions().MetricsScope().
+		SubScope("dbindex")
```

Comment:
could you add a tag for namespace name here too

Reply:
Ah yes, sure thing.